components/net/http_loader.rs

```diff
@@ -694,6 +694,11 @@ pub fn http_redirect_fetch(
         },
         Some(Ok(url)) => url,
     };
+    context
```

Comment:
Let's add a comment here like `// Step 1 of https://w3c.github.io/resource-timing/#dom-performanceresourcetiming-fetchstart`